src/ut/alarm_tables_testing.cpp

```diff
@@ -63,10 +63,11 @@ char buf[1024];
 
 TEST_F(CustomDefs, ModelTable)
 { 
-  AlarmTableDefs::get_instance().insert_def(*def_cleared);
-  AlarmTableDefs::get_instance().insert_def(*def_raised);
+  AlarmTableDefs* alarm_table_defs = new AlarmTableDefs();
```

Comment:
Why not just `AlarmTableDefs alarm_table_defs`? It doesn't look like it needs to be on the heap and that saves you a line of cleanup code.